utils/lib/Alert.ts

```diff
@@ -51,16 +57,25 @@ class Alert extends Component<QuickstartConfigAlert[], QuickstartAlertInput[]> {
         return yaml.load(file.toString('utf-8')) as QuickstartConfigAlert;
       });
     } catch (e) {
-      console.log('Unable to parse YAML config', this.configPath, e);
+      console.log(`Unable to parse YAML config for alert: ${this.configPath}`);
       this.isValid = false;
 
       return this.config;
     }
   }
 
   getMutationVariables() {
-    if (!this.isValid) {
-      return [];
+    const isPathGiven = this.configPath === ''
```

Comment:
Shouldn't the Alert be considered invalid if the config path is the empty string?